docs/rules.md

```diff
@@ -49,6 +49,12 @@ hide:
     11. Do not write or install software on computers belonging to other players or servers without their informed consent.
         1.  When advertising malicious software, it should be made very clear that it is malicious during every interaction.
     12. Do not use No Chat Reports' chat encryption feature in public chat. Using it in private messages is allowed however, as long as the receiving party is able to decode it.
+    13. Any form of in game gambling must be fair to the players.
+        1. Gambling must *never* be done with (currency bought with) money that has value in the real world (FIAT, Crypto, etc.)
```

Comment:
Critical thinking in english tells you putting something behind in parenthesis makes it a specifier, not an inclusive clause.

```Gambling must *never* involve money that has value in the real world (FIAT, Crypto, etc.) directly or indirectly.```